### ISSUE TITLE
hang during build of unicode_xid inside travis CI

### ISSUE THREAD

**Issue by Dushistov** [mozilla/sccache#226]
I got series of failure with sccache (0.2.5) it looks like this:

```
Compiling unicode-xid v0.0.4
Running `sccache rustc --crate-name unicode_xid /home/travis/.cargo/registry/src/github.com-1ecc6299db9ec823/unicode-xid-0.0.4/src/lib.rs --crate-type lib --emit=dep-info,metadata -C debuginfo=2 --cfg 'feature="default"' -C metadata=7b951564ce49c932 -C extra-filename=-7b951564ce49c932 --out-dir /home/travis/build/Dushistov/rust_swig/target/debug/deps -L dependency=/home/travis/build/Dushistov/rust_swig/target/debug/deps --cap-lints allow`


No output has been received in the last 10m0s, this potentially indicates a stalled build or something wrong with the build itself.
Check the details on how to adjust your build configuration on: https://docs.travis-ci.com/user/common-build-problems/#Build-times-out-because-no-output-was-received

The build has been terminated
```

See [full log of build](https://api.travis-ci.org/v3/job/345469354/log.txt)

I see this only for beta (and some times nightly) during `cargo doc`.

At first I clean all caches in Travis CI, you can see that:
```
du -s -c -h /home/travis/.cache/sccache
0	/home/travis/.cache/sccache
0	total
```
But this not helps, build still fails.

After that I removed `    - export RUSTC_WRAPPER=sccache` from `.travis.yml` and build become green.

**Comment by luser** [comment]
Try again with 0.2.6? We fixed an issue with the jobserver support that was added in 0.2.5.

**Comment by luser** [comment]
I'm going to preemptively close this assuming it was fixed by the changes in 0.2.6. If you can reproduce with 0.2.6 please reopen!
